app/views/maps/_form.html.erb

```diff
@@ -19,5 +19,8 @@
   <div class="flex items-center justify-between">
     <%= form.submit class: "btn" %>
     <%= link_to "Cancel", "#", data: { action: "click->modal#close" }, class: "inline-block align-baseline text-sm text-gray-500 hover:text-gray-800" %>
+    <% if map.persisted? %>
+      <%= link_to "Delete", map_path(map), method: :delete, remote: true, data: { confirm: "Are you sure you want to delete #{map.name}?" }, class: "inline-block align-baseline text-sm text-red-500 hover:text-red-800" %>
+    <% end %>
   </div>
```

Comment:
The buttons/links currently spread out evenly which looks a little odd to me:
<img width="468" alt="Screenshot 2020-05-31 at 09 52 17" src="https://user-images.githubusercontent.com/1191840/83348431-6bbe3880-a324-11ea-858e-bb33bd17a7ed.png">

A div around two of them would group them together nicely:
<img width="456" alt="Screenshot 2020-05-31 at 09 55 57" src="https://user-images.githubusercontent.com/1191840/83348479-f010bb80-a324-11ea-9f2c-cbf308311004.png">

```suggestion
  <div class="flex items-center justify-between">
    <div>
      <%= form.submit class: "btn" %>
      <% if map.persisted? %>
        <%= link_to "Delete", map_path(map), method: :delete, remote: true, data: { confirm: "Are you sure you want to delete #{map.name}?" }, class: "inline-block align-baseline text-sm text-red-500 hover:text-red-800 px-2" %>
      <% end %>
    </div>
    <%= link_to "Cancel", "#", data: { action: "click->modal#close" }, class: "inline-block align-baseline text-sm text-gray-500 hover:text-gray-800" %>
  </div>
```